code/modules/mob/living/carbon/human/examine.dm

```diff
@@ -46,11 +46,10 @@
 		if(!(I.item_flags & ABSTRACT) && !(I.item_flags & EXAMINE_SKIP))
 			. += "[t_He] [t_is] holding [I.get_examine_string(user)] in [t_his] [get_held_index_name(get_held_index_of_item(I))]."
 
-	var/datum/component/forensics/FR = GetComponent(/datum/component/forensics)
 	//gloves
 	if(gloves && !(obscured & ITEM_SLOT_GLOVES) && !(gloves.item_flags & EXAMINE_SKIP))
 		. += "[t_He] [t_has] [gloves.get_examine_string(user)] on [t_his] hands."
-	else if(FR && length(FR.blood_DNA))
+	else if(forensics && length(forensics.blood_DNA))
```

Comment:
this should've been using the helper getters anyways
doesn't matter much now, but might as well enforce it for consistency(?)

```suggestion
	else if(return_blood_DNA())
```
or
```suggestion
	else if(blood_DNA_length())
```